Tests/Functional/WebhookHandler/WebhookResource.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * (c) shopware AG <info@shopware.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace SwagPaymentPayPalUnified\Tests\Functional\WebhookHandler;
+
+class WebhookResource
```

Comment:
could you rename it to something, so it contains "test" or so? 

so it gets not confused with the https://github.com/shopware5/SwagPaymentPayPalUnified/blob/main/PayPalBundle/Resources/WebhookResource.php

Reply:
Done